package.json

```diff
@@ -1,36 +1,45 @@
 {
   "name": "labs-authorization-service",
   "version": "0.0.2",
-  "description": "Full stack authorization Service",
+  "description": "A policy based authorization module",
+  "license": "MIT",
+  "author": "nearForm Ltd",
+  "contributors": [
+    "Dean McDonnell (https://github.com/mcdonnelldean)",
+    "Mihai Dima (https://github.com/mihaidma)",
+    "Paolo Chiodi (https://github.com/paolochiodi)"
```

Comment:
Forgot about Nicholas. Also from previous team there is Mark Ireland, Michele. Not sure about the guys that worked on front end Salman, Micael O'Brien